### PR TITLE
add graphql fields to query schedules/sensors by status

### PR DESCRIPTION
**Pull request by prha**
## Summary & Motivation
GraphQL fields to filter sensors/schedules by status.  Resolves the stored status `AUTOMATICALLY_RUNNING` to simply `RUNNING`.

## How I Tested These Changes
BK

### CODE REVIEW

**Comment by prha** [comment]
Current dependencies on/for this PR:
* master
  * **PR #13212** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13212" target="_blank">Graphite</a> 
    * **PR #13213** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13213" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13213?utm_source=stack-comment).

File: python_modules/dagster-graphql/dagster_graphql/implementation/fetch_schedules.py [hunk 1]
```diff
@@ -109,25 +112,62 @@ def get_schedules_or_error(
...
-    schedule_states_by_name = {
-        state.name: state
-        for state in graphene_info.context.instance.all_instigator_state(
+    if schedule_status == InstigatorStatus.RUNNING:
+        schedule_states = graphene_info.context.instance.all_instigator_state(
             repository_origin_id=repository.get_external_origin_id(),
             repository_selector_id=repository_selector.selector_id,
             instigator_type=InstigatorType.SCHEDULE,
+            instigator_status=InstigatorStatus.RUNNING,
+        ) + graphene_info.context.instance.all_instigator_state(
```
Comment (alangenfeld) [inline python_modules/dagster-graphql/dagster_graphql/implementation/fetch_schedules.py:121]: [1]

File: python_modules/dagster-graphql/dagster_graphql/implementation/fetch_sensors.py [hunk 2]
```diff
@@ -38,22 +45,60 @@ def get_sensors_or_error(
...
-    sensor_states_by_name = {
-        state.name: state
-        for state in graphene_info.context.instance.all_instigator_state(
+    if sensor_status == InstigatorStatus.RUNNING:
+        sensor_states = graphene_info.context.instance.all_instigator_state(
+            repository_origin_id=repository.get_external_origin_id(),
+            repository_selector_id=repository_selector.selector_id,
+            instigator_type=InstigatorType.SENSOR,
+            instigator_status=InstigatorStatus.RUNNING,
+        ) + graphene_info.context.instance.all_instigator_state(
```
Comment (alangenfeld) [inline python_modules/dagster-graphql/dagster_graphql/implementation/fetch_sensors.py:54]: [1]

**Review by alangenfeld — COMMENTED** [review]
[1] should we use a separate enum at the GraphQL layer that we do the mapping with? Its a bit confusing to resolve to `InstigatorStatus` and then do the special treatment.

File: python_modules/dagster-graphql/dagster_graphql/implementation/fetch_schedules.py [hunk 3]
```diff
@@ -99,7 +100,9 @@ def get_scheduler_or_error(graphene_info: ResolveInfo) -> "GrapheneScheduler":
 
 @capture_error
 def get_schedules_or_error(
-    graphene_info: ResolveInfo, repository_selector: RepositorySelector
+    graphene_info: ResolveInfo,
+    repository_selector: RepositorySelector,
+    schedule_status: Optional[InstigatorStatus] = None,
```
Comment (gibsondan) [inline python_modules/dagster-graphql/dagster_graphql/implementation/fetch_schedules.py:105]: should we call it instigator_status

**Review by gibsondan — DISMISSED** [review]
for alex's feedback which seems reasonable to me

**Review by gibsondan — APPROVED** [review]

**Pull request merged**